Addresses issue with validateReportingStructure

colinfrederickson opened the pull request:
- now clears messages from employeeId field when running function
- now returns all records, not just those with issues

Accounts for scenarios where data is updated and the function is re-run, messages are now accurate.

colinfrederickson committed:
Addresses issue with validateReportingStructure

- now clears messages from employeeId field when running function
- now returns all records, not just those with issues

Accounts for scenarios where data is updated and the function is re-run, messages are now accurate.

johnmosesman commented on actions/validateReportingStructure.js:
```diff
@@ -89,5 +95,5 @@ export const validateReportingStructure = (records) => {
   console.log('Reporting Errors: ' + JSON.stringify(reportingErrors));
 
-  return reportingErrors;
+  return records;
```
Can you convert to .ts?

johnmosesman reviewed (approved):
@colinfrederickson one note on .ts, otherwise merge conflict and lgtm

colinfrederickson committed:
Optimize validateReportingStructure function

Modified the validateReportingStructure function to return only modified records. This optimization reduces the amount of unnecessary data returned and streamlines the validation process. Removed redundant checks and streamlined error messages for better clarity.

validateReportingStructure to .ts

Bumped @flatfile/listener to 0.3.13

colinfrederickson committed:
Merge branch 'main' into addresses-issue-with-validateReportingStructure

colinfrederickson commented:
@johnmosesman - this should be good to go!

johnmosesman committed:
Undo package lock changes
